vllm/distributed/device_communicators/custom_all_reduce_utils.py

```diff
@@ -213,7 +212,12 @@ def gpu_p2p_access_check(src: int, tgt: int) -> bool:
         # However, `can_actually_p2p` requires spawn method.
         # The fix is, we use `subprocess` to call the function,
         # where we have `if __name__ == "__main__":` in this file.
-        input_bytes = pickle.dumps((batch_src, batch_tgt))
+
+        # use a temporary file to store the result
+        # we don't use the output of the subprocess directly,
+        # because the subprocess might produce logging output
+        output_file = tempfile.NamedTemporaryFile(delete=False)
```

Comment:
Could you use this as a context manager ... down to after where the pickle.load is done.. then it will clean up the file automatically I think

Reply:
fixed in [afb0855](https://github.com/vllm-project/vllm/pull/7853/commits/afb0855bdc69a2e45e02fb32b9ad6c5ce2510bbd)